js/modules/k6/http/request.go

```diff
@@ -437,7 +437,14 @@ func (c *Client) prepareBatchObject(requests map[string]interface{}) (
 
 // Batch makes multiple simultaneous HTTP requests. The provideds reqsV should be an array of request
 // objects. Batch returns an array of responses and/or error
-func (c *Client) Batch(reqsV goja.Value) (interface{}, error) {
+func (c *Client) Batch(reqsValues ...goja.Value) (interface{}, error) {
+	if len(reqsValues) == 0 {
+		return nil, fmt.Errorf("no argument was provided to http.batch()")
+	} else if len(reqsValues) > 1 {
+		return nil, fmt.Errorf("http.batch() must have only one single array as an argument")
+	}
```

Comment:
If we add a specific validation for this case then it makes sense to add a test case for it. Can you add one, please?

Reply:
Hello @codebien 

Thanks a lot  for mentioning the same. I have added the tests in request_test.go.
I have also accepted the CLA as mentioned. 
https://cla-assistant.io/grafana/k6?pullRequest=2585

Very thankful to you for checking my contribution .

Regards
Vanshaj